code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -419,6 +419,7 @@ GLOBAL_LIST_INIT(titanium_recipes, list(
 GLOBAL_LIST_INIT(plastitanium_recipes, list(
 	new /datum/stack_recipe("plas-titanium tile", /obj/item/stack/tile/mineral/plastitanium, 1, 4, 20),
 	new /datum/stack_recipe("Kidan Warrior Statue", /obj/structure/statue/plastitanium/kidanstatue, 5, time = 2.5 SECONDS, one_per_turf = TRUE, on_floor = TRUE),
+	new /datum/stack_recipe("Reinforced Wheelchair", /obj/structure/chair/wheelchair/plastitanium, 15, time = 60 SECONDS, one_per_turf = TRUE),
```

Comment:
60 seconds for reinforced and 20 for hardened? Thats not too much?

Reply:
well they are intentionally this long as for wheelchairs, i can decrease time if its too much?